src/components/ShowcaseCards.astro

```diff
@@ -0,0 +1,157 @@
+  {
+    title: "rebel-in-the-sky",
+    owner: "ricott1",
+    repo: "rebels-in-the-sky",
+    url: "https://github.com/ricott1/rebels-in-the-sky",
+    description: "P2P terminal game about spacepirates",
+    media:
+      "https://www.youtube.com/embed/5Lu9MSgmTBc?autoplay=1&loop=1&playlist=5Lu9MSgmTBc&mute=1&controls=0&modestbranding=0&rel=0&showinfo=0",
+    alt: "rebel-in-the-sky demo",
+  },
+  {
+    title: "binsider",
+    owner: "orhun",
+    repo: "binsider",
+    url: "https://github.com/orhun/binsider",
+    description: "Perform binary analysis in your terminal",
+    media:
+      "https://github.com/orhun/binsider/blob/main/website/src/content/assets/quickstart.gif?raw=true",
+    alt: "binsider demo",
+  },
+  {
+    title: "crossword",
+    owner: "matrixfrog",
+    repo: "crossword",
+    url: "https://github.com/matrixfrog/crossword",
+    description: "Play crossword puzzles in your terminal",
+    media: crosstui.src,
+    alt: "crossword demo",
+  },
+  {
+    title: "yozefu",
+    owner: "MAIF",
+    repo: "yozefu",
+    url: "https://github.com/MAIF/yozefu",
+    description: "Explore Kafka clusters interactively",
+    media: "https://vhs.charm.sh/vhs-UpIJD2h92vKkj01XSS0r0.gif",
+    alt: "yozefu demo",
+  },
+  {
+    title: "openai-tui",
+    owner: "zaghaghi",
+    repo: "openapi-tui",
+    url: "https://github.com/zaghaghi/openapi-tui",
+    description: "Terminal UI to list, browse and run APIs defined with OpenAPI",
+    media: "https://github.com/zaghaghi/openapi-tui/blob/main/static/demo.gif?raw=true",
+    alt: "openai-tui demo",
+  },
+  {
+    title: "csvlens",
+    owner: "YS-L",
+    repo: "csvlens",
+    url: "https://github.com/YS-L/csvlens",
+    description: "A command line CSV file viewer",
+    media: "/csvlens.gif",
+    alt: "csvlens demo",
+  },
+  {
+    title: "oxker",
+    owner: "mrjackwills",
+    repo: "oxker",
+    url: "https://github.com/mrjackwills/oxker",
+    description: "View & control docker containers with a TUI",
+    media: "https://raw.githubusercontent.com/mrjackwills/oxker/main/.github/demo_01.webp?raw=true",
+    alt: "oxker demo",
+  },
+];
+
+const tileData = [3, 2, 1, 1, 3, 2, 1, 1];
+---
+
+<div class="not-content">
+  <h2
+    class="flex items-start pt-[4rem] pb-6 text-4xl font-bold text-black max-xl:justify-center max-sm:pt-[50px] dark:text-white"
+  >
+    <span class="pr-2"
+      >Built with <span class="text-blue-700 dark:text-yellow-500">Ratatui</span></span
+    >
+    <img class="m-0 hidden h-8 md:block" src={chef.src} />
+    <img class="m-0 hidden h-8 pl-2 md:block" src={rat.src} />
+  </h2>
+  <p class="max-w-[1440px] pb-12 text-lg leading-8 max-xl:text-center max-sm:text-sm">
+    Ratatui has all the ingredients you need to cook up exceptional terminal applications. Always
+    fresh and full of cheese.
+  </p>
+  <div class="m-auto flex max-w-[1440px] flex-wrap gap-6">
+    {
+      projects.map((project: ShowcaseProject, index: number) => (
+        <div
```

Comment:
I'd suggest pushing this div down into the card so that the the styling is there as a single element rather than two elements - pass the tileData[index] === 1 bit as a flag to the compoent instead.

Same with the vids